assets/stylesheets/layout/_main.scss

```diff
@@ -130,10 +130,15 @@ body.rtl,
 	display: inherit;
 }
 
+html {
+	overflow: none;
```

Comment:
fyi `none` is not a valid value for `overflow`
you may want to use `visible` or remove this rule completely